packages/block-library/src/image/block.json

```diff
@@ -9,9 +9,6 @@
 	"keywords": [ "img", "photo", "picture" ],
 	"textdomain": "default",
 	"attributes": {
-		"align": {
-			"type": "string"
-		},
```

Comment:
I went ahead and added a commit to remove this since it gets added by the block supports. The only difference should be the serialization order moving the align attribute to the block support section after the custom attribute section.